bindings/wasm/bindings.js

```diff
@@ -248,8 +248,12 @@ Module.setup = function() {
     return this._Translate(vararg2vec3(vec));
   };
 
-  Module.Manifold.prototype.rotate = function(vec) {
-    return this._Rotate(...vec);
+  Module.Manifold.prototype.rotate = function(xOrVec, y, z) {
+    if (Array.isArray(xOrVec)) {
+      return this._Rotate(...xOrVec);
```

Comment:
I believe we already have a helper function you can use here: `vararg2vec3`.

Reply:
I was considering using `vararg2vec3` however `this._Rotate` seems to be a bit different from `_Scale`, `_Mirror` or `_Translate`—it doesn't take `glm::vec3` but just three floats. So we'd have to do:
```ts
Module.Manifold.prototype.rotate = function(...v) {
  const vec = vararg2vec3(v)
  return this._Rotate(vec.x, vec.y, vec.z);
};
```
which seems wasteful—we are creating an object just to destructure it immediately. Perhaps it's a negligible performance hit, so if you like this version more, let me know and I will change it in the PR

Reply:
Okay, that's fine, thanks.